### DISCUSSION TITLE
Working with custom slotfills

### DISCUSSION THREAD

**Discussion by ryanwelcher** [WordPress/developer-blog-content#140] [Topic Ideas]
This would be a follow up article to [How to extend WordPress via the SlotFill system](https://developer.wordpress.org/news/2022/11/how-to-extend-wordpress-via-the-slotfill-system/) where we would show how to create a custom SlotFill and use it in an existing codebase.

**Comment by justintadlock** [comment]
I like it. This is an area that's a bit of a mystery to me, mostly because I've never dove into it. I'd love to see a post that covers this.

**Comment by bph** [comment]
The topic has been approved. The discussion is now locked,
Please continue the conversation on the issue
- #146

**Comment by ryanwelcher** [comment]
Published - https://developer.wordpress.org/news/2023/12/22/extending-plugins-using-custom-slotfills/
